poseidon-paramgen/src/matrix.rs

```diff
@@ -23,13 +23,15 @@ pub use traits::{MatrixOperations, SquareMatrixOperations};
 mod tests {
```

Comment:
this module would make much more sense in `poseidon-parameters` (both traits and implementors are there)

Reply:
moved to tests.rs module